src/main/resources/db/migration/V6_2__alter_table_motedeltaker_behandler_varsel_svar.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE MOTEDELTAKER_BEHANDLER_VARSEL_SVAR ADD COLUMN msg_id TEXT;
```

Comment:
Bør vi sette en definert lengde her, f.eks. uuid-lengde, eller kan vi ikke vite?

Reply:
Ja, jeg tenkte at det kan være greit å ta høyde for at disse strengene kan være lange. Dok'en sier bare at det skal være en GUID.